lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -130,12 +127,14 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
   defp create_block_response_chunk({:ok, block}) do
     with {:ok, ssz_signed_block} <- Ssz.to_ssz(block),
          {:ok, snappy_ssz_signed_block} <- Snappy.compress(ssz_signed_block) do
+      fork_context = BeaconChain.get_fork_digest()
```

Comment:
We should add a comment on how this doesn't take into account inter-fork requests. If we ask for blocks of an earlier hard-fork, we will fail because our fork digest uses the newer fork version.

Reply:
yes, make sense.